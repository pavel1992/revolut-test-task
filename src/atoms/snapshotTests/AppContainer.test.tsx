import 'jest-styled-components';
import * as React from 'react';
import renderer from 'react-test-renderer';

import { AppContainer } from "../AppÐ¡ontainer";


test('renders app container correctly', () => {
  const render = renderer
    .create(<AppContainer/>)
    .toJSON();
  expect(render).toMatchSnapshot();
});
