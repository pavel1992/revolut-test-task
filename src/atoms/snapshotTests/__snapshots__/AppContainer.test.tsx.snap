// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders app container correctly 1`] = `
.c0 {
  width: 100%;
  height: 100%;
  font-style: normal;
  font-weight: normal;
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-flex-direction: row;
  -ms-flex-direction: row;
  flex-direction: row;
  -webkit-box-pack: center;
  -webkit-justify-content: center;
  -ms-flex-pack: center;
  justify-content: center;
}

<div
  className="c0"
/>
`;
